src/OpenTK.Mathematics/MathHelper.cs

```diff
@@ -67,6 +67,69 @@ public static class MathHelper
         /// </summary>
         public const float Log2E = 1.442695041f;
 
+        /// <summary>
+        /// Returns the sine of the specified angle.
+        /// </summary>
+        /// <param name="a">An angle, measured in radians.</param>
+        /// <returns>Sine of the angle. If a is equal to NaN, NegativeInfinity, or PositiveInfinity, this method returns NaN.</returns>
+        public static double Sin(double a) => Math.Sin(a);
+
+        /// <summary>
+        /// Returns the hyperbolic sine of the specified angle.
+        /// </summary>
+        /// <param name="a">An angle, measured in radians.</param>
+        /// <returns>Hyperbolic sine of the specified angle. If a is equal to NaN, NegativeInfinity, or PositiveInfinity, this method returns NaN.</returns>
+        public static double HSin(double a) => Math.Sinh(a);
+
+        /// <summary>
+        /// Returns the arc sine of the specified angle.
+        /// </summary>
+        /// <param name="a">An angle, measured in radians.</param>
+        /// <returns>Arc sine of the specified angle. If a is equal to NaN, NegativeInfinity, or PositiveInfinity, this method returns NaN.</returns>
+        public static double ASin(double a) => Math.Asin(a);
+
+        /// <summary>
+        /// Returns the cosine of the specified angle.
+        /// </summary>
+        /// <param name="a">An angle, measured in radians.</param>
+        /// <returns>Cosine of the angle. If a is equal to NaN, NegativeInfinity, or PositiveInfinity, this method returns NaN.</returns>
+        public static double Cos(double a) => Math.Cos(a);
+
+        /// <summary>
+        /// Returns the hyperbolic cosine of the specified angle.
+        /// </summary>
+        /// <param name="a">An angle, measured in radians.</param>
+        /// <returns>Hyperbolic cosine of the specified angle. If a is equal to NaN, NegativeInfinity, or PositiveInfinity, this method returns NaN.</returns>
+        public static double HCos(double a) => Math.Cosh(a);
```

Comment:
Cosh here(see above HSin->Sinh).

Reply:
I mostly did this to try and go with a style i saw better fit, however if there is a mathemathical reason for these names or it is more consistent with what is used elsewhere im of cause going to change it. 
It is changed in the most recent comit

Reply:
Fixed in most recent commit